src/types/events.ts

```diff
@@ -127,12 +127,15 @@ export type OnAudioFocusChangedData = Readonly<{
 
 export type OnBufferData = Readonly<{isBuffering: boolean}>;
 
-export type OnBandwidthUpdateData = Readonly<{
-  bitrate: number;
-  width: number;
-  height: number;
-  trackId: number;
-}>;
+export type OnBandwidthUpdateData = Readonly<
+  | {
+      bitrate: number;
+      width: number;
```

Comment:
same here width?: number;

Reply:
But here i would like to leave it like this, it will be easier for user - if width is not null it means that height is also not null. Correct me if I am wrong

Reply:
ok, no problem, and yes you are right, if wifth is ull height is null also